sdk/cognitivelanguage/azure-ai-language-questionanswering/tests/test_query_knowledgebase_async.py

```diff
@@ -317,9 +320,9 @@ async def test_query_knowledgebase_with_followup(self, qna_account, qna_key, qna
             )
 
             assert len(output.answers) == 2
-            confident_answers = [a for a in output.answers if a.confidence_score > 0.6]
+            confident_answers = [a for a in output.answers if a.confidence_score > 0.5]
             assert len(confident_answers) == 1
-            assert confident_answers[0].answer_span.text == "two to four hours"
+            assert confident_answers[0].answer_span.text == " two to four hours"
```

Comment:
intentional spacing?

Reply:
yeah sadly the spacing the service returns is different